domain/notifier.go

```diff
@@ -6,11 +6,13 @@ type NotificationMessages struct {
 	AppealRejected       string `mapstructure:"appeal_rejected" default:"Your appeal to {{.resource_name}} with role {{.role}} has been rejected"`
 	AccessRevoked        string `mapstructure:"access_revoked" default:"Your access to {{.resource_name}}} with role {{.role}} has been revoked"`
 	ApproverNotification string `mapstructure:"approver_notification" default:"You have an appeal created by {{.requestor}} requesting access to {{.resource_name}} with role {{.role}}. Appeal ID: {{.appeal_id}}"`
+	OthersAppealApproved string `mapstructure:"other_appeal_approved" default:"Your appeal to {{.resource_name}} with role {{.role}} created by {{.requestor}} has been approved"`
 }
 
 const (
 	NotificationTypeExpirationReminder   = "ExpirationReminder"
 	NotificationTypeAppealApproved       = "AppealApproved"
+	NotificationTypeAppealForOnBehalf    = "OnBehalfAppealApproved"
```

Comment:
```suggestion
	NotificationTypeOnBehalfAppealApproved    = "OnBehalfAppealApproved"
```

Reply:
I think it is better to have consistent naming for const var and string on this @bsushmith @singhvikash11

Reply:
Yes @mabdh. This is the updated line as per the suggestion1. Did not commit the suggestion through github UI., as there was one other place it required renaming. 

```go
	NotificationTypeOnBehalfAppealApproved = "OnBehalfAppealApproved"

```